Library/Formula/rstudio-server.rb

```diff
@@ -0,0 +1,49 @@
+class RstudioServer < Formula
+  homepage "http://www.rstudio.com"
+  url "https://github.com/rstudio/rstudio/archive/v0.98.1103.tar.gz"
+  sha256 "084049aae03cbaaa74a848f491d57cffab5ea67372ece0e256d54e04bd5fc6da"
+
+  depends_on "ant" => :build
+  depends_on "cmake" => :build
+  depends_on "r" => :build
+
+  def install
+    # patch the boost install scripts
+    inreplace "dependencies/common/install-boost", "/opt/rstudio-tools/boost",
+      "#{prefix}/boost"
+    inreplace "dependencies/common/install-boost", "sudo ", ""
+    inreplace "src/cpp/CMakeLists.txt", "/opt/rstudio-tools/boost", "#{prefix}/boost"
+
+    # specific path to R explicitly
+    inreplace "dependencies/common/install-packages", "R CMD build",
```

Comment:
Can you please submit these patches to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!